src/LanguageServer/Protocol/Handler/Diagnostics/AbstractPullDiagnosticHandler.cs

```diff
@@ -116,15 +116,15 @@ protected virtual Task WaitForChangesAsync(string? category, RequestContext cont
             var clientCapabilities = context.GetRequiredClientCapabilities();
             var category = GetRequestDiagnosticCategory(diagnosticsParams);
             var handlerName = $"{this.GetType().Name}(category: {category})";
-            context.TraceInformation($"{handlerName} started getting diagnostics");
+            context.TraceDebug($"{handlerName} started getting diagnostics");
 
             var versionedCache = _categoryToVersionedCache.GetOrAdd(
                 handlerName, static (handlerName, globalOptions) => new(globalOptions, handlerName), GlobalOptions);
 
             // Get the set of results the request said were previously reported.  We can use this to determine both
             // what to skip, and what files we have to tell the client have been removed.
             var previousResults = GetPreviousResults(diagnosticsParams) ?? [];
-            context.TraceInformation($"previousResults.Length={previousResults.Length}");
+            context.TraceDebug($"previousResults.Length={previousResults.Length}");
```

Comment:
Not really related to this, but whenever I saw this in the logs it was very unclear what this was related to. Not sure if it should also be prefixed with handlerName or something.

Reply:
it already is prefixed by the method name
![image](https://github.com/user-attachments/assets/4e1cb904-153a-4dee-b952-5c16ebe7923f)